database/seed.js

```diff
@@ -156,7 +157,7 @@ const seedDb = async () => {
       newBooking = {
         campId: 0,
         price_per_night: 165,
-        booked: [[1, 2, 3], [15, 16, 17], [25, 26, 27], [28, 29]],
+        booked: [[6], [1, 2, 3], [15, 16, 17], [25, 26, 27], [28, 29]],
```

Comment:
What does this nested array represent?

Reply:
it is the month that the booked dates fall under